sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
 # Release History
 
 ## 1.2.8-beta.1 (Unreleased)
+### Breaking Changes
+- Update `com.microsoft.azure` group id to `com.azure.spring`.
```

Comment:
Maybe "Change group id from `com.microsoft.azure` to `com.azure.spring`." is better